### PR TITLE
UX improvements

### PR DESCRIPTION
**Pull request by elbayaaa**
*Issue #, if available:*
N/A

*Description of changes:*
Some UX improvements

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by aahung — APPROVED** [review]

File: GitHub-Actions/two-environment-pipeline-template/questions.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
   "questions": [{
+    "key": "intro",
+    "question": "\nThis template configures a pipeline that deploys a serverless application to a testing and a production environments.\n",
```
Comment (hoffa) [inline GitHub-Actions/two-environment-pipeline-template/questions.json:4]:
```suggestion
    "question": "\nThis template configures a pipeline that deploys a serverless application to a testing and a production environment.\n",
```

File: Gitlab/two-environment-pipeline-template/questions.json [hunk 2]
```diff
@@ -1,5 +1,9 @@
 {
   "questions": [{
+    "key": "intro",
+    "question": "\nThis template configures a pipeline that deploys a serverless application to a testing and a production environments.\n",
```
Comment (hoffa) [inline Gitlab/two-environment-pipeline-template/questions.json:4]:
```suggestion
    "question": "\nThis template configures a pipeline that deploys a serverless application to a testing and a production environment.\n",
```

File: GitHub-Actions/two-environment-pipeline-template/questions.json [hunk 3]
```diff
@@ -1,5 +1,9 @@
...
+    "kind": "info"
+  }, {
     "key": "pipeline_user_aws_access_key_id_variable_name",
     "question": "What is the GitHub Secret name for pipeline user account access key id?",
```
Comment (hoffa) [inline GitHub-Actions/two-environment-pipeline-template/questions.json:8]:
```suggestion
    "question": "What is the GitHub secret name for pipeline user account access key ID?",
```

File: Gitlab/two-environment-pipeline-template/questions.json [hunk 4]
```diff
@@ -1,5 +1,9 @@
...
+    "kind": "info"
+  }, {
     "key": "main_git_branch",
     "question": "What is the git branch used for production deployments?",
```
Comment (hoffa) [inline Gitlab/two-environment-pipeline-template/questions.json:8]:
```suggestion
    "question": "What is the Git branch used for production deployments?",
```

File: Gitlab/two-environment-pipeline-template/{{cookiecutter.outputDir}}/.gitlab-ci.yml [hunk 5]
```diff
@@ -145,6 +137,15 @@ deploy-testing:
                  --no-fail-on-empty-changeset
                  --role-arn ${TESTING_CLOUDFORMATION_EXECUTION_ROLE}
 
+# uncomment and modify the following step for running the integration-tests
```
Comment (hoffa) [inline Gitlab/two-environment-pipeline-template/{{cookiecutter.outputDir}}/.gitlab-ci.yml:140]:
```suggestion
# Uncomment and modify the following stage for integration tests
```

File: Jenkins/two-environment-pipeline-template/questions.json [hunk 6]
```diff
@@ -1,5 +1,9 @@
 {
   "questions": [{
+    "key": "intro",
+    "question": "\nThis template configures a pipeline that deploys a serverless application to a testing and a production environments.\n",
```
Comment (hoffa) [inline Jenkins/two-environment-pipeline-template/questions.json:4]:
```suggestion
    "question": "\nThis template configures a pipeline that deploys a serverless application to a testing and a production environment.\n",
```

**Review by hoffa — CHANGES_REQUESTED** [review]

File: GitHub-Actions/two-environment-pipeline-template/questions.json (hunk 3 above)
Comment (hoffa) [reply GitHub-Actions/two-environment-pipeline-template/questions.json:8]: Actually this applies to all instances:

- Should be "GitHub secrets"; it's not used as a proper noun.
- ID should be all caps.
- Git should be capitalized (not "git").

**Review by xazhao — APPROVED** [review]
LGTM!

**Review by hoffa — APPROVED** [review]
LGTM; we should still do the following before release:

- Should be "GitHub secret"; it's not used as a proper noun.
- ID should be all caps.
- Git should be capitalized (not "git").

**Pull request merged**